test/controllers/FileChecksResultsControllerSpec.scala

```diff
@@ -150,6 +158,75 @@ class FileChecksResultsControllerSpec extends FrontEndTestHelper {
         status(recordCheckResultsPage) mustBe 200
         contentType(recordCheckResultsPage) mustBe Some("text/html")
 
+        checkPageForStaticElements.checkContentOfPagesThatUseMainScala(resultsPageAsString, userType = userType)
+        resultsPageAsString must include(expectedTitle)
+        resultsPageAsString must include(expectedHeading)
+        if (userType == "standard") {
+          resultsPageAsString must include(expectedSuccessSummaryTitle)
+          resultsPageAsString must include(confirmTransferButton)
+        }
+
+        if (userType == "judgment") {
+          resultsPageAsString must include regex (buttonToProgress)
+        }
+
+        resultsPageAsString must include(expectedSuccessMessage)
+      }
+
+      s"render the $userType fileChecksResults page with the confirmation box" in {
+        val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+        val consignmentService = new ConsignmentService(graphQLConfiguration)
+        val consignmentStatusService = new ConsignmentStatusService(graphQLConfiguration)
+
+        val config = ConfigFactory
+          .load()
+          .withValue("featureAccessBlock.closureMetadata", ConfigValueFactory.fromAnyRef("false"))
+          .withValue("featureAccessBlock.descriptiveMetadata", ConfigValueFactory.fromAnyRef("false"))
+        setConsignmentStatusResponse(app.configuration, wiremockServer)
+        val fileStatus = List(gfcp.GetConsignment.Files(Some("Success")))
+
+        val fileChecksData = gfcp.Data(
+          Option(
+            GetConsignment(allChecksSucceeded = true, Option("parentFolder"), 1, fileStatus, FileChecks(AntivirusProgress(1), ChecksumProgress(1), FfidProgress(1)))
+          )
+        )
+
+        val filePathData = gcf.Data(
+          Option(gcf.GetConsignment(List(Files(UUID.randomUUID(), Option(""), Option(""), Option(UUID.randomUUID()), Metadata(Some("test file.docx"))))))
+        )
+
+        val getFileChecksProgressClient = graphQLConfiguration.getClient[gfcp.Data, gfcp.Variables]()
+        val getConsignmentFilesClient = graphQLConfiguration.getClient[gcf.Data, gcf.Variables]()
+        val fileStatusResponse: String =
+          getFileChecksProgressClient.GraphqlData(Option(fileChecksData), List()).asJson.printWith(Printer(dropNullValues = false, ""))
+        val filePathResponse: String =
+          getConsignmentFilesClient.GraphqlData(Option(filePathData), List()).asJson.printWith(Printer(dropNullValues = false, ""))
+
+        mockGraphqlResponse(userType, fileStatusResponse, filePathResponse)
+        setConsignmentReferenceResponse(wiremockServer)
+
+        val fileCheckResultsController = new FileChecksResultsController(
+          getAuthorisedSecurityComponents,
+          keycloakConfiguration,
+          new GraphQLConfiguration(app.configuration),
```

Comment:
You can use the graphQLConfiguration used on line 177. Ideally we can extract out the instantiation of the FileChecksResultsController to a private method as this would reduce the amount code in this test spec but that's for another time.

Reply:
The configuration is passed in a class constructor parameter so it needs to be created anyway.